docs/Privacy/Tessera/Usage/Logging.md

```diff
@@ -0,0 +1,71 @@
+Entity not found: {error details} | API request received against q2tserver/transaction/{key} where key is not a tx hash in the DB.<br>
+Entity not found:{error details} | Thrown if endpoint doesn't exist on that API, e.g.<br>    curl -s http://localhost:9001/invalidendpoint<br>
+Security exception {followed by exception message, like "java.lang.SecurityException: No key found for url 127.1.1.1"} | Thrown if 'enableRemoteKeyValidation' is true and partyinfo request received from a URL for which we don't hold a public key (i.e. potentially a malicious party).<br>Note: if key validation enabled then this exception will be thrown during startup whilst the nodes exchange key information.
+ERROR c.q.t.a.e.DefaultExceptionMapper - HTTP 400 Bad Request | Logged if received message is corrupt/incorrectly formatted, along with the warning below, e.g.<br>    curl -X POST "http://localhost:9001/resend" -H "accept: text/plain" -H "Content-Type: application/json" -d "{ \"some rubbish\" }"<br>
+Error while reading secret from file | Unable to read the secret key (password) from file specified by TESSERA_CONFIG_SECRET.<br>Action: ensure the secret key file config is correct, and file can be read
+unable to initialize encryption façade {error details} | Unable to initialise elliptical curve encryption. Logged error message will give further details.<br>Action: check configuration properties
+unable to generate shared secret {error details} | Unable to generate shared secret for elliptical curve encryption. Logged error message will give further details.<br>Action: check configuration properties
+unable to perform symmetric encryption {error details} | Unable to encrypt data. Logged error message will give further details.<br>Action: check configuration properties
+unable to perform symmetric decryption {error details} | Unable to decrypt data. Logged error message will give further details.<br>Action: check configuration properties
+Error when executing action {action type}, exception details: {error details} | Unable to start Influx DB. Logged error message will give further details.<br>Action: check configuration properties
+Error creating bean with name 'entityManagerFactory' | Unable to create connection to database due to failure to decrypt the DB password using the supplied secret key.<br>Action: ensure that the correct value is supplied for the secret key.
+Config validation issue: {property name} {error details} | Invalid configuration detected.<br>Action: correct the configuration of the named property.
+Invalid json, cause is {error details} | Invalid json in the configuration file.<br>Action: check the configuration file for mistakes.
+Configuration exception, cause is {error details} | Invalid data in the configuration file.<br>Action: check the configuration file for mistakes.
+CLI exception, cause is {error details} | Invalid command line.<br>The error details will give further information regarding the action to be taken. 
+
+## Warnings
```

Comment:
same comment as above to convert the table to a 3 column one with distinct action

Reply:
Adding a third column has a negative impact on the formatting, however the steps have been emphasised so that they are easier to identify